modules/rules_kotlin/1.9.0/MODULE.bazel

```diff
@@ -0,0 +1,71 @@
+module(
+    name = "rules_kotlin",
+    version = "1.9.0",
+    compatibility_level = 1,
+    repo_name = "rules_kotlin",
+)
+
+bazel_dep(name = "platforms", version = "0.0.6")
+bazel_dep(name = "bazel_skylib", version = "1.4.2")
+bazel_dep(name = "rules_java", version = "6.4.0")
+bazel_dep(name = "rules_python", version = "0.23.1")
+bazel_dep(name = "rules_cc", version = "0.0.8")
+
+rules_kotlin_extensions = use_extension("//src/main/starlark/core/repositories:bzlmod_setup.bzl", "rules_kotlin_extensions")
+use_repo(
+    rules_kotlin_extensions,
+    "buildkite_config",
+    "com_github_google_ksp",
+    "com_github_jetbrains_kotlin",
+    "com_github_pinterest_ktlint",
+    "kt_java_stub_template",
+    "rules_android",
+)
+
+register_toolchains("//kotlin/internal:default_toolchain")
+
+# TODO(bencodes) We should be able to remove this once rules_android has rolled out official Bzlmod support
+remote_android_extensions = use_extension("@bazel_tools//tools/android:android_extensions.bzl", "remote_android_tools_extensions")
+use_repo(remote_android_extensions, "android_gmaven_r8", "android_tools")
+
+# Development dependencies
+
+bazel_dep(name = "rules_jvm_external", version = "5.3")
```

Comment:
these development dependencies aren't marked as `dev_dependency=True`. is that intentional?

Reply:
@Wyverald This was intentional.

I have a few more use cases to sort out around how we are packaging up `rules_kotlin` before I can safely start marking these as development dependencies. We'll start removing some of these in future releases as we sort things out.

Reply:
I see. would be nice if you have a small comment here explaining this, but we don't have to block this first release on that.

Reply:
This release was tagged from a working branch. I'll add some notes before we merge everything.